core/sources/dnsdb.go

```diff
@@ -0,0 +1,68 @@
+package sources
+
+import (
+	"bufio"
+	"net"
+	"net/http"
+	"regexp"
+	"strings"
+	"time"
+
+	core "github.com/subfinder/research/core"
+)
+
+type DnsDbDotCom struct {
+	Name string
+}
+
+// NewDNSDbDotCom creates a new instance of DnsDbDotCom
+func NewDNSDbDotCom() *DnsDbDotCom {
```

Comment:
I have some reservations about this function, only because it's only acting as a initializer to set the `Name` of this source when no other source is currently implemented that way. ( to be clear, I could be open to that ) 

If there were more sources that needed initialization this way, I would think this is great, but it might be a smidgen too much. But I appreciate the work! ❤️I simply think it would be simpler for this case if the source was initialized like any other source, at least for now.